app/templates/manage/placement/form.hbs

```diff
@@ -28,6 +28,30 @@
       {{/type-ahead}}
     </div>
   </div>
+  <div class="col">
+    <div class="form-group">
+      <label for="placement-reservePct">Impression Reserve</label>
```

Comment:
I know, I know, you hate me... change `for` and `id` to dasherized `placement-reserve-pct`